test/releasers/terraform-module.ts

```diff
@@ -50,16 +51,23 @@ describe('terraform-module', () => {
         'versions.tf',
         'modules/sub-module-with-version/versions.tf',
       ],
+      findReadmeFiles: [
+        'readme.md',
+        // 'module-submodule/modules/sub-module-missing-versions/README.md',
+        'module-submodule/modules/sub-module-with-version/readme.md'
+      ],
       readFilePaths: [
         'module-submodule/readme.md',
         'module-submodule/versions.tf',
+        // 'module-submodule/modules/sub-module-missing-versions/README.md',
```

Comment:
All readme.md files would be read first as we have the readmeFiles update before the versionFiles update. Also missing `'module-submodule/modules/sub-module-with-version/readme.md',`
```ts
      readFilePaths: [
        'module-submodule/readme.md',
        'module-submodule/modules/sub-module-missing-versions/README.md',
        'module-submodule/modules/sub-module-with-version/readme.md',
        'module-submodule/versions.tf',
        'module-submodule/modules/sub-module-with-version/versions.tf',
      ]
```